R/Model-class.R

```diff
@@ -1145,7 +1145,7 @@ DualEndpoint <- function(mean,
   assert_numeric(rho, min.len = 1, max.len = 2)
 
   use_fixed <- c(sigma2W = is.scalar(sigma2W), rho = is.scalar(rho))
-  betaZ_params <- ModelParamsNormal(mean, cov)
+  betaZ_params <- ModelParamsNormal(mean, cov) # nolintr
```

Comment:
ok because I guess we have `betaZ` as parameter so we don't want to change that

Reply:
Exactly.